README.md

```diff
@@ -296,6 +330,28 @@ Make sure that your `$GOBIN` is in your `$PATH`.
    protoc -I . --openapiv2_out ./gen/openapiv2 --openapiv2_opt logtostderr=true your/service/v1/your_service.proto
```

Comment:
You are referring to `protoc` but at the same time, the paragraph at the top about installing it is removed.

Reply:
I reordered these, but I think I'm okay with removing the note at the top about how to install protoc, since it is no longer _required_.